impeller/geometry/half.h

```diff
@@ -0,0 +1,176 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <cstdint>
+
+#include "flutter/fml/build_config.h"
+
+#include "color.h"
+#include "point.h"
+#include "scalar.h"
+#include "vector.h"
```

Comment:
Includes should be rooted at the base of the project as seen above: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
Done